sample/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,17 @@
+<resources xmlns:tools="http://schemas.android.com/tools">
+
+    <style name="Theme.LintSample" parent="Theme.MaterialComponents.DayNight.DarkActionBar">
+        <item name="colorPrimary">@color/primary</item>
+        <item name="colorPrimaryVariant">@color/primary_variant</item>
+        <item name="colorOnPrimary">@color/color_on_primary</item>
+        <item name="colorSecondary">@color/secondary</item>
+        <item name="colorSecondaryVariant">@color/secondary_variant</item>
+        <item name="colorOnSecondary">@color/color_on_secondary</item>
+        <item name="android:statusBarColor" tools:targetApi="21">?attr/colorPrimaryVariant</item>
+    </style>
+
+    <style name="Widget.Button" parent="Widget.MaterialComponents.Button" />
+
+    <style name="ShapeAppearanceOverlay" />
```

Comment:
again, we can move this to its file named `redumdant_style_sample.xml` and name the style itself `redundantStyleSample` with a comment explaining it

Reply:
Done.